### DISCUSSION TITLE
How to provide multiple sources to Next Image component

### DISCUSSION THREAD

**Discussion by alex-mngt** [vercel/next.js#28426] [Help] [answered]
I'm pretty sure what I ask is not yet possible but I open this discussion to be sure I won't open a feature request which is already implemented. 

I would like to give the Image component multiple src just like in a native html srcset attribute in order to display different images ( not just cropped ) depending on the viewport. For example a Ubuntu logo on desktop, a MacOS one on tablet and a Windows on phone.

**Comment by leerob** [comment, marked as answer]
Correct, this is not supported currently. I am going to lock this and move the discussion to the RFC for `picture`, which would support this.

https://github.com/vercel/next.js/discussions/25393
